crates/dojo-world/src/manifest/types.rs

```diff
@@ -94,9 +94,14 @@ pub struct DojoContract {
     #[serde_as(as = "UfeHex")]
     pub base_class_hash: FieldElement,
     pub abi: Option<AbiFormat>,
+    #[serde(default)]
     pub reads: Vec<String>,
+    #[serde(default)]
     pub writes: Vec<String>,
+    #[serde(default)]
     pub computed: Vec<ComputedValueEntrypoint>,
+    #[serde(default)]
+    pub constructor_calldata: Vec<String>,
```

Comment:
```suggestion
    pub init_calldata: Vec<String>,
```
I suggest keeping the same term all over the stack? We should refer to `init_calldata` as it's not strictly a constructor to avoid confusion.

wdyt?